### PR TITLE
UIU-1793: Refactoring of pay fee/fine single action

### PR DESCRIPTION
**Pull request by annamelnyk**
# Description
- Refactoring of pay fee/fine single action
- Refactoring of charge&pay fee/fine single action
The main goal of refactoring to remove fee/fine calculation's logic to BE.
2 endpoint were added:-
- `/accounts/{accountId}/check-pay`
- `/accounts/{accountId}/pay`

# Tasks
https://issues.folio.org/browse/UIU-1793
https://issues.folio.org/browse/UIU-1836

### CODE REVIEW

**Review by zburke — APPROVED** [review]

File: src/components/Accounts/Actions/ActionModal.js [hunk 1]
```diff
@@ -167,14 +147,75 @@ class ActionModal extends React.Component {
...
-  onChangeOwner = () => {
-    const { dispatch } = this.props;
-    dispatch(change('payment-many-modal', 'method', null));
+  triggerCheckEndpoint = (amount, accountId) => {
+    const {
+      checkAmount,
+      okapi,
+    } = this.props;
+
+    return fetch(`${okapi.url}/accounts/${accountId}/${checkAmount}`,
```
Comment (mkuklis) [inline src/components/Accounts/Actions/ActionModal.js:161]: @annamelnyk I'm wondering did you run into some limitations with `stripes-connect` here and had to do a direct `fetch`?

**Review by mkuklis — APPROVED** [review]
Nice work @annamelnyk. I left one little question.

File: src/components/Accounts/Actions/ActionModal.js (hunk 1 above)
Comment (annamelnyk) [reply src/components/Accounts/Actions/ActionModal.js:161]: This request was implemented by @maximdidenko. As I understand direct fetch needed for continuous validation and calculations of `remaining amount` on server side while `amount` is changing in `ActionModal`

File: src/components/Accounts/Actions/FeeFineActions.js [hunk 2]
```diff
@@ -261,13 +308,48 @@ class Actions extends React.Component {
...
+    const account = _.head(accounts) || {};
+    mutator.activeRecord.update({ id: account.id });
+    const payload = this.buildActionBody(values);
+    mutator[this.actionToEndpointMapping[action]].POST(_.omit(payload, ['id']))
       .then(() => this.props.handleEdit(1))
-      .then(() => this.showCalloutMessage(type))
+      .then(() => this.showCalloutMessage(account))
       .then(() => this.onCloseActionModal());
+
+    // delete type.rowIndex;
```
Comment (maximdidenkoepam) [inline src/components/Accounts/Actions/FeeFineActions.js:348]: Please remove commented code

**Review by maximdidenkoepam — APPROVED** [review]

**Pull request merged**